scripts/microbe_stage/setup.as

```diff
@@ -130,7 +130,7 @@ void setupPlayer(CellStageWorld@ world)
     GetThriveGame().playerData().lockedMap().addLock("chloroplast");
 
     ObjectID microbe = MicrobeOperations::spawnMicrobe(world, Float3(0, 0, 0), "Default",
-        false);
+        false, false);
```

Comment:
Having a default value for partOfColony would be nice to not have these changes.

Reply:
Just another question, I don't really understand what you mean by default value here. Should I override the function with a version that doesn't take this parameter? Or do you not need to enter all the parameters in angelscript?

Reply:
I'm pretty sure you can set default parameter values the same way in AS as in C++.

Reply:
Alright thanks! I've made the requested changes, this should be good now.